### ISSUE TITLE
Project status? 

### ISSUE THREAD

**Issue by vladdu** [pawamoy/dependenpy#50]
Hi, is the project maintained? The last commit was almost 2 years ago, and there are many things that would have to be updated to keep up with the times (python versions, dependency versions)

**Comment by pawamoy** [comment]
Hello! It's maintained yes! It's just that it's kind of feature-complete now so I don't spend time on it.
I can try to update things a bit :smile:

**Comment by pawamoy** [comment]
Yeah, I had updates sitting on my disk for a long time. Currently fixing the setup, should be ready soon.

**Comment by vladdu** [comment]
I have some feature additions that I implemented, for example to sort the DSM matrix so that it is triangular; and some that I need, for example to check if there are dependency cycles, and to suggest better clustering of modules. It would be useful to extend this to the function level too, for more detailed analysis (But then a text.matrix is hard to look at)

**Comment by pawamoy** [comment]
That's great! Contributions are absolutely welcome :smile: Project setup should be fixed now :slightly_smiling_face:

**Comment by vladdu** [comment]
Excellent, thanks!
